doc/educational_decoder/Makefile

```diff
@@ -26,7 +26,10 @@ test: harness
 	@./harness tmp.zst tmp
 	@diff -s tmp README.md
 	@$(RM) -f tmp*
-	@zstd --train harness.c zstd_decompress.c zstd_decompress.h README.md
+	# present files for training multiple times, to reach minimum threshold
+	@zstd --train harness.c zstd_decompress.c zstd_decompress.h README.md \
```

Comment:
This only works if `zstd` is installed on the system. We should pass a `ZSTD` variable to this script.